src/rules/javascript/webextension-unsupported-api.js

```diff
@@ -5,7 +5,8 @@ export default {
   create(context) {
     return {
       MemberExpression: function(node) {
-        if (node.object.object &&
+        if (!node.computed &&
```

Comment:
This test is mostly here as a convenience to developers from what I understand. If they want to try and use `chrome.gcm.send()` they can try it but we don't support it.

I'm not sure this is something that would need to extra review. If there are certain APIs that need extra validation then we can check those but that likely belongs somewhere else.